src/main/java/com/github/wovnio/wovnjava/Api.java

```diff
@@ -155,6 +158,10 @@ private URL getApiUrl(String lang, String body) throws UnsupportedEncodingExcept
         appendValue(sb, lang);
         appendValue(sb, "&version=wovnjava_");
         appendValue(sb, Settings.VERSION);
+        if (headers.getCacheDisableMode() || headers.getDebugMode()) {
```

Comment:
I don't think you necessarily want to do this for only debug mode

Reply:
I believe we talked about only allowing cache bypass if debugMode was turned on server-side

Reply:
that is built-in in the `getCacheDisableMode` check isn't it? But just enabling debug mode you might want to use the cached version until you decide to override it.